src/sw.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference lib="WebWorker" />
+// Empty export required due to --isolatedModules flag in tsconfig.json
+export type {};
+declare const self: ServiceWorkerGlobalScope;
+declare const clients: Clients;
+
+const assets = ["simulator.html", "build/simulator.js", "build/firmware.js"];
```

Comment:
Should firmware.wasm be here? It's loaded via fetch by the sim JS.

Reply:
Yes, it should. Done in [2063919](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/115/commits/2063919a3156d7363c8e8ffe0a6518b26558a3d8).